static/css/app.css

```diff
@@ -289,7 +327,7 @@ md-data-table {
   md-data-table .md-data-table-header-column.sorted {
     color: rgba(0, 0, 0, 0.87); }
   md-data-table .md-data-table-row-cell {
-    color: rgba(0, 0, 0, 0.87);
+    color: #243058;
```

Comment:
leave it black, considering the specs says it should be black, should be overriden somewhere else